lnwallet/chanfunding/assembler.go

```diff
@@ -80,6 +80,13 @@ type Request struct {
 	// e.g. anchor channels.
 	WalletReserve btcutil.Amount
 
+	// Outpoints is a list of client-selected outpoints that should be used
+	// for funding a channel. If LocalAmt is specified then this amount is
+	// allocated from the sum of outpoints towards funding. If the
+	// FundUpToMaxAmt is specified the entirety of selected funds is
+	// allocated towards channel funding.
+	Outpoints []*wire.OutPoint
```

Comment:
I think can have just be regular values (no pointer?), outpoint are pretty small as is (and can simplify validation elsewhere).